docs/source/user_documentation/installation.rst

```diff
@@ -6,47 +6,58 @@ Installation
 
 ``skbase`` currently supports:
 
-* environments with python version 3.7, 3.8, 3.9 or 3.10
+* environments with python version 3.7, 3.8, 3.9, 3.10 or 3.11
 * operating systems Mac OS X, Unix-like OS, Windows 8.1 and higher
 
 Checkout the full list of pre-compiled wheels on
-`PyPI <https://pypi.org/simple/baseobject/>`_.
+`PyPI <https://pypi.org/simple/skbase/>`_.
 
 Release versions
 ================
 
 Most users will be interested in installing a released version of ``skbase``
 using one of the approaches outlined below. For common installation issues,
-see the `Release versions - troubleshooting`_ section.
+see the `troubleshooting release installations`_ section.
 
-Installing ``skbase`` from PyPI
--------------------------------
+Installing ``skbase``
+---------------------
 
-``skbase`` releases are available via PyPI and can be installed via ``pip`` using:
+``skbase`` releases are available via PyPI and can be installed via ``pip``. Users
+can choose whether to install the ``skbase`` with its standard dependencies or
+alternatively to install ``skbase`` with all its dependencies using the
+code snippets below.
 
-.. code-block:: bash
+.. tab-set::
 
-    pip install skbase
+    .. tab-item:: PyPi
 
-This will install ``skbase`` with core dependencies, excluding soft dependencies.
+        .. code-block:: bash
 
-To install ``skbase`` with maximum dependencies, including soft dependencies,
-install with the ``all_extras`` modifier:
+           pip install skbase
 
-.. code-block:: bash
+    .. tab-item:: PyPi (all dependencies)
+
+        .. code-block:: bash
+
+           pip install skbase[all_extras]
```

Comment:
there's not much interesting in `all_extras` - only `pandas`, and most users will have that installed anyway. Is it still worth highlighting?

Reply:
I don't think it hurts.